Install postgres

wamts opened the pull request.

wamts committed:
Ran npm -install

wamts committed:
Installed pg, sequelize node packages

wamts committed:
Installed postgres and sequelize packages

skassam21 commented on .gitignore:
```diff
@@ -20,4 +20,4 @@ yarn-error.log*
 
 
 #TODO add .env to your gitignore
```
can remove this from TODO now

skassam21 commented on server/config/config.json:
```diff
@@ -0,0 +1,26 @@
+    "dialect": "postgres",
+    "operatorsAliases": false
+  }
+}
```
This should be coming from your env variables

wamts replied:
Updated to come from config.js, via process.env.DB_[VALUE]

skassam21 commented on server/models/user.js:
```diff
@@ -0,0 +1,12 @@
+  const User = sequelize.define('User', {
+    firstName: DataTypes.STRING,
+    lastName: DataTypes.STRING,
+    email: DataTypes.STRING
```
I noticed that there is createdAt and updatedAt in migrations and not here - do you not need access to them

d-ang1993 replied:
So I played around with the **sequelize-cli** today 
Even though we don't have an createdAt & updatedAt within our mock model, the **sequelize-cli** will generate a createdAt & updatedAt column within our postgresql database.

Anytime we create a **User** in this case, those fields will auto-populate. 
I'm not sure if this answers this comment, but we do have access to them. 

But if best practice is to include the createdAt and updatedAt within the model, we can add a **timestamp** option field each time we create the model using **sequelize-cli**. (But I'm not sure if this will affect it in anyway, something we can test out :smile:)

wamts replied:
Yeah, it might not be totally necessary to include createdAt and updatedAt in the user schema.

I've added new fields in user.js anyways in the latest commit but its always easy to go back and update them.

skassam21 commented:
@wamts @DangAsian I assigned @kidistz, @kidistz you will want to look through this and try to run it

kidistz reviewed (approved):
Review good.

wamts committed:
Replaced config.json with config.js

Added .sequelizerc to allow use of config.js

user.js schema is now updated to include create and update time

wamts committed:
Fixed merge conflict with dev branch

wamts committed:
Removed comment from .gitignore

d-ang1993 reviewed (approved):
Everything looks good to go :+1:

wamts committed:
Close #6 gitignored .env file

wamts committed:
Merge branch 'dev' into install-postgres
